domains/certificates/Shared/RabbitMqEvents/EnergyMeasuredIntegrationEvent.cs

```diff
@@ -1,4 +1,4 @@
-namespace IntegrationEvents;
+namespace RabbitMqEvents;
```

Comment:
Have you considered other names than RabbitMqEvents? I was thinking something like "MeasurementEvents" or "MeasurementMessages" as this is releated to measurements.

Reply:
renamed to MeasurementEvents